.github/workflows/aws-terraform.yml

```diff
@@ -4,42 +4,37 @@ on:
   workflow_call:
     inputs:
       docker_image:
-        description: 'The runner image to be used'
+        description: "The runner image to be used"
         required: true
         type: string
       github_environment:
-        description: 'Defines the github environemnt for secrets'
+        description: "Defines the GitHub environment for secrets"
         required: true
         type: string
       deployment_environment:
-        description: 'Defines the deployment environemnt for makefile'
+        description: "Defines the deployment environment for Makefile"
         required: true
         type: string
       tf_action:
-        description: 'Terraform action to perform'
+        description: "Terraform action to perform"
         required: true
         type: string
       tf_backend_bucket_name:
-        description: 'Terraform backend bucket Name'
-        required: true
+        description: "Terraform backend bucket Name"
```

Comment:
please update description with details if we keep this var empty

Reply:
Updated in f7270e29dddbc6b96c720aaca7302044b7b08e54